runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -189,6 +233,108 @@ private static long getSubscriptionTimestamp(CorfuStore corfuStore, String names
             MicroMeterUtils.time(subscribeTimer, "logreplication.subscribe.duration");
         }
     }
+    private static long getRoutingQueueSubscriptionTimestamp(CorfuStore corfuStore, String namespace,
```

Comment:
This is the same as `getSubscriptionTimestamp`...why not use the same?
`    private static long getSubscriptionTimestamp(CorfuStore corfuStore, String namespace,
                                                 StreamListener clientListener) {}`

Reply:
No, there are some changes in these two methods. So, kept it separate.